utilities/transfer/dashboard_export.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+
+OPTSPEC=":hu:p:t:f:"
+
+show_help() {
+cat << EOF
+Usage: $0 [-u USER] [-p PASSWORD] [-f FROM_FOLDER] [-t TARGET_HOST]
```

Comment:
Trivial style comment.  Consider moving all of the required before the optional (put -t before -f):
Usage: $0 [-u USER] [-p PASSWORD] [-t TARGET_HOST] [-f FROM_FOLDER]

Reply:
Done